recognition/include/pcl/recognition/ransac_based/bvh.h

```diff
@@ -192,11 +192,8 @@ namespace pcl
             inline bool
             intersect(const float box[6]) const
             {
-              if ( box[1] < bounds_[0] || box[3] < bounds_[2] || box[5] < bounds_[4] ||
-                   box[0] > bounds_[1] || box[2] > bounds_[3] || box[4] > bounds_[5] )
-                return false;
-
-              return true;
+              return !static_cast<bool>(box[1] < bounds_[0] || box[3] < bounds_[2] || box[5] < bounds_[4] ||
```

Comment:
Why `static_cast` here?

Reply:
Good question, no idea why clang-tidy has done this.

Reply:
I found more of this unnecessary `static_cast<bool>` in code base, so I added a new commit instead of modifying existing one, so it is clear this changes are not automatically done.

Reply:
Great, thanks!